examples/image-type-r-dataframe.yaml

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: spark.apache.org/v1alpha1
+kind: SparkApplication
+metadata:
+  name: sparkr-dataframe
+spec:
+  driverSpec:
+    podTemplateSpec:
+      metadata:
+      spec:
+        containers:
+          - name: driver
+            workingDir: /opt/spark
+  executorSpec:
+    podTemplateSpec:
+      metadata:
+      spec:
+        containers:
+          - name: executor
+            workingDir: /opt/spark
+  sparkRFiles: "local:///opt/spark/examples/src/main/r/dataframe.R"
```

Comment:
Ditto. We don't want to advertise `dataframe.R` because R is deprecated.
- https://github.com/apache/spark/pull/47842